gdk/static/user_input_recipe_schema.json

```diff
@@ -0,0 +1,353 @@
+{
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
```

Comment:
Are these properties case-insensitive?

Reply:
Based on my experience, the case doesn't matter for the recipe properties as long as the property name is accurate.